cmd/api/src/api/v2/auth/saml.go

```diff
@@ -17,24 +17,23 @@
 package auth
 
 import (
-	"context"
 	"encoding/xml"
 	"errors"
 	"fmt"
+	"io"
 	"net/http"
+	"strconv"
 
 	"github.com/crewjam/saml"
 	"github.com/crewjam/saml/samlsp"
 	"github.com/gorilla/mux"
-	"github.com/specterops/bloodhound/crypto"
 	"github.com/specterops/bloodhound/headers"
 	"github.com/specterops/bloodhound/log"
 	"github.com/specterops/bloodhound/mediatypes"
 	"github.com/specterops/bloodhound/src/api"
-	saml2 "github.com/specterops/bloodhound/src/api/saml"
-	"github.com/specterops/bloodhound/src/auth/bhsaml"
-	"github.com/specterops/bloodhound/src/config"
-	bhCtx "github.com/specterops/bloodhound/src/ctx"
+	v2 "github.com/specterops/bloodhound/src/api/v2"
```

Comment:
Is the `v2` in front of `"github.com/specterops/bloodhound/src/api/v2"` needed when it already ends with `v2`?